cmd/awscollector/main.go

```diff
@@ -48,8 +47,22 @@ func main() {
 	// init cfgFactory
 	cfgFactory := config.GetParserProvider()
 
+	var zapHooks []zap.Option
+
 	// init lumberFunc for zap logger
 	lumberHook := logger.GetLumberHook()
+	if lumberHook != nil {
+		zapHooks = append(zapHooks, zap.Hooks(lumberHook))
+	}
+
+	ecsErrorLogFilePath := os.Getenv("ECS_ERROR_LOG_FILE_PATH")
+	// init an ECS Error Logger and a go routine for error reporting when the STATUS_MESSAGE_FILE_PATH is set
+	if ecsErrorLogFilePath != "" {
```

Comment:
Since you only use `ecsErrorLogFilePath` to check if the logger is enable, you can write `if os.Getenv("ECS_ERROR_LOG_FILE_PATH") != ""` here, also it might be better to write a method in the `logger` package like `logger.ECSErrorLogEnabled()` to avoid writing the constant string `ECS_ERROR_LOG_FILE_PATH` in two places.